src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -305,6 +305,59 @@ private async Task ReplyOnReAuthenticationAsync<TIOAdapter>(TIOAdapter adapter,
             }
         }
 
+        // This will initiate renegotiation or PHA for Tls1.3
+        private async Task Renegotiate(CancellationToken cancellationToken)
```

Comment:
RenegotiateAsync, since it's an async method?

Reply:
updated.